landing/src/main.tsx

```diff
@@ -10,11 +10,13 @@ import SignIn from './signIn/SignIn'
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
     <React.StrictMode>
         <HashRouter>
-            <Header />
-            <Routes>
-                <Route path='/' element={<Landing />} />
-                <Route path='/innlogging' element={<SignIn />} />
-            </Routes>
+            <div className="min-h-screen flex flex-col">
+                <Header />
+                <Routes>
+                    <Route path="/" element={<Landing />} />
+                    <Route path="/innlogging" element={<SignIn />} />
+                </Routes>
+            </div>
```

Comment:
La til dette elementet for å bruke flex-box til å strekke elementet under `<Header>` til full høyde i nettleservinduet uten å hardkode verdier.